test/unit/strategies/extensions/RecipientsExtension.t.sol

```diff
@@ -240,16 +240,32 @@ contract RecipientsExtensionReviewRecipients is BaseRecipientsExtensionUnit {
 }
 
 contract RecipientsExtensionReviewRecipientStatus is BaseRecipientsExtensionUnit {
```

Comment:
I'm not sure about the refactor of this 2 tests. @ilpepepig your opinion is appreciated.

Reply:
LGTM